### PR TITLE
support popular craft values and more office values

### LINKED ISSUE #1525: support craft=tailor and other craft=* for OH, wheelchair and name quests
I will do it after #1522 (assuming that supporting craft=* will not be rejected)

Noticed thanks to Vespucci validator.

### PR DESCRIPTION
**Pull request by matkoniecz**
fixes #1525

### CODE REVIEW

**Comment by westnordost** [comment]
Why would painters, window construction workshops, caterers, hvac, blacksmiths, upholsterer etc. have no name? Summarizing:
- if the place is open for walk-in by general public -> ask for opening hours
- if the place is either the above or at least public in the frame of public events -> ask for wheelchair accessibility
- if the place like this usually or at least sometimes has a name -> ask for a name

I don't think your categorization with the OsmTaggings works here. Probably, almost any "craft" thing will have a name. In the case of crafts, the list for opening hours and wheelchair accessibility should almost be the same.

**Comment by westnordost** [comment]
Actually, it is the question whether not much more if not almost all `office` keys would usually have a name

**Comment by matkoniecz** [comment]
I pushed some commits, but are untested (and tag list would benefit from fresh look)

File: app/src/main/java/de/westnordost/streetcomplete/quests/place_name/AddPlaceName.kt [hunk 1]
```diff
@@ -48,7 +48,27 @@ class AddPlaceName(
...
+                // also listed for AddOpeningHours and AddWheelchair quests
+                "carpenter", "shoemaker", "tailor", "photographer", "dressmaker",
+                "electronics_repair", "key_cutter", "stonemason",
+                // also listed for AddPlaceName quest
+                "winery", "electrician", "plumber",
+                // ones below not listed for other quests
+                "metal_construction", "confectionery", "caterer", "hvac", "window_construction",
+                "handicraft", "gardener", "painter", "glaziery", "beekeper", "blacksmith",
+                "roofer", "upholsterer", "pottery"
+            )
```
Comment (westnordost) [inline app/src/main/java/de/westnordost/streetcomplete/quests/place_name/AddPlaceName.kt]: or maybe any `craft`? The same is also done for `shop` further up.

File: app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt [hunk 2]
```diff
@@ -33,7 +33,19 @@ class AddWheelchairAccessBusiness(o: OverpassMapDataDao) : SimpleOverpassQuestTy
...
+                "insurance", "government", "travel_agent", "tax_advisor", "religion", "employment_agency",
+                // also listed for AddPlaceName quest
+                "lawyer", "estate_agent", "political_party", "therapist"
+            ),
+            "craft" to arrayOf(
+                // also listed for AddOpeningHours and AddPlaceName quests
+                "carpenter", "shoemaker", "tailor", "photographer", "dressmaker",
+                "electronics_repair", "key_cutter", "stonemason",
+                // also listed for AddPlaceName quest
+                "winery", "electrician", "plumber"
```
Comment (westnordost) [inline app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt]: Not sure why you include plumber and electrician. Wouldn't you then also need to include hvac (etc etc)?

**Comment by matkoniecz** [comment]
I am thinking about excluding shop/craft=vacant/no

This problem is basically not appearing for craft and quite rare for office so I decided to not include this checks.

**Comment by westnordost** [comment]
Ok, looks good!

**Pull request merged**